api-server/server/utils/constants.js

```diff
@@ -83,5 +84,537 @@ export const blacklistedUsernames = [
+  'new',
+  'news',
+  'newsletter',
+  'newsletters',
+  'next',
+  'nil',
+  'no-reply',
+  'nobody',
+  'noc',
+  'none',
+  'noreply',
+  'notification',
+  'notifications',
+  'ns',
+  'ns0',
+  'ns1',
+  'ns2',
+  'ns3',
+  'ns4',
+  'ns5',
+  'ns6',
+  'ns7',
+  'ns8',
+  'ns9',
+  'null',
+  'oauth',
+  'oauth2',
+  'offer',
+  'offers',
+  'online',
+  'openid',
+  'order',
+  'orders',
+  'overview',
+  'owner',
+  'page',
+  'pages',
+  'partners',
+  'passwd',
+  'password',
+  'pay',
+  'payment',
+  'payments',
+  'photo',
+  'photos',
+  'pixel',
+  'plans',
+  'plugins',
+  'policies',
+  'policy',
+  'pop',
+  'pop3',
+  'popular',
+  'portfolio',
+  'post',
+  'postfix',
+  'postmaster',
+  'poweruser',
+  'preferences',
+  'premium',
+  'press',
+  'previous',
+  'pricing',
+  'print',
+  'privacy',
+  'privacy-policy',
+  'private',
+  'prod',
+  'product',
+  'production',
+  'profile',
+  'profiles',
+  'project',
+  'projects',
+  'public',
+  'purchase',
+  'put',
+  'quota',
+  'redirect',
+  'reduce',
+  'refund',
+  'refunds',
+  'register',
+  'registration',
+  'remove',
+  'replies',
+  'reply',
+  'report',
+  'request',
+  'request-password',
+  'reset',
+  'reset-password',
+  'response',
+  'return',
+  'returns',
+  'review',
+  'reviews',
+  'robots.txt',
+  'root',
+  'rootuser',
+  'rsa-sha2-2',
+  'rsa-sha2-512',
+  'rss',
+  'rules',
+  'sales',
+  'save',
+  'script',
+  'sdk',
+  'search',
+  'secure',
+  'security',
+  'select',
+  'services',
+  'session',
+  'sessions',
+  'settings',
+  'setup',
+  'share',
+  'shift',
+  'shop',
+  'signin',
+  'signup',
+  'site',
+  'sitemap',
+  'sites',
+  'smtp',
+  'sort',
+  'source',
+  'sql',
+  'ssh',
+  'ssh-rsa',
+  'ssl',
+  'ssladmin',
+  'ssladministrator',
+  'sslwebmaster',
+  'stage',
+  'staging',
+  'stat',
+  'static',
+  'statistics',
+  'stats',
+  'status',
+  'store',
+  'style',
+  'styles',
+  'stylesheet',
+  'stylesheets',
+  'subdomain',
+  'subscribe',
+  'sudo',
+  'super',
+  'superuser',
+  'support',
+  'survey',
+  'sync',
+  'sysadmin',
+  'system',
+  'tablet',
+  'tag',
+  'tags',
+  'team',
+  'telnet',
+  'terms',
+  'terms-of-use',
+  'test',
+  'testimonials',
+  'theme',
+  'themes',
+  'today',
+  'tools',
+  'topic',
+  'topics',
+  'tour',
+  'training',
+  'translate',
+  'translations',
+  'trending',
+  'trial',
+  'true',
+  'umac-128',
+  'umac-128-etm',
+  'umac-64',
+  'umac-64-etm',
+  'undefined',
+  'unfollow',
+  'unlike',
+  'unsubscribe',
+  'update',
+  'upgrade',
+  'usenet',
+  'user',
+  'username',
+  'users',
+  'uucp',
+  'var',
+  'verify',
+  'video',
+  'view',
+  'void',
+  'vote',
+  'webmail',
+  'webmaster',
+  'website',
+  'widget',
+  'widgets',
+  'wiki',
+  'wpad',
+  'write',
+  'www',
+  'www-data',
+  'www1',
+  'www2',
+  'www3',
+  'www4',
+  'you',
+  'yourname',
+  'yourusername',
+  'zlib'
 ];
+
+export const blocklistedUsernames = blocklist.filter((value, index, self) => {
```

Comment:
The current filter combined with indexOf  may not be as efficient as the following:
```suggestion
export const blocklistedUsernames = [ ...new Set(blocklist) ];
```